pontoon/sync/core/checkout.py

```diff
@@ -99,6 +99,11 @@ def checkout_repos(
     """
     source: Checkout | None = None
     target: Checkout | None = None
+
+    # If the project using configuration file has unsynced locales, perform forced sync
+    if project.configuration_file:
+        force = force or project.unsynced_locales
```

Comment:
Better to avoid the query if `force` is already true, and keep `force` as a boolean value rather than a queryset.
```suggestion
    if not force and project.configuration_file and project.unsynced_locales:
        force = True
```